setup/service/service.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func RestartPostgres() error {
+	// TODO: account for different environments; handle error output
```

Comment:
Do we need to resolve this TODO for the initial release?

Reply:
That's a good question. For different envs, looks like Debian's been using systemd by default since 8, the oldest supported release, so we're fine there. I'll double-check the error output handling.